docs/compression_algorithms/CompressWeights.md

```diff
@@ -293,6 +293,8 @@ Here is the word perplexity with data-free and data-aware mixed-precision INT4-I
 - [LLM Weight Compression](https://docs.openvino.ai/nightly/weight_compression.html)
 - [Optimize and Deploy Generative AI Models using Hugging Face Optimum Intel](https://docs.openvino.ai/nightly/gen_ai_guide.html)
 - [Optimum Intel documentation](https://huggingface.co/docs/optimum/intel/inference)
+- [Large Language Models Weight Compression Example](https://github.com/openvinotoolkit/nncf/blob/develop/examples/llm_compression/openvino/tiny_llama)
+- [Tuning Ratio and Group Size Example](https://github.com/openvinotoolkit/nncf/blob/develop/examples/llm_compression/openvino/tiny_llama_find_hyperparams)
```

Comment:
Thanks for adding the link to the example! 
I think it would be useful to elaborate more on tuning of hyperparams. For example, after words:
>If the accuracy or perplexity is still not satisfying, there are 2 more hyper-parameters to tune: `group_size` and `ratio`.

Reply:
Now it looks out of the context. 

I meant a different thing. This sentence is already in the section with example. Please mention your script after that sentence there with some quick overview. e.g.
```
If the accuracy or perplexity is still not satisfying, there are 2 more hyper-parameters to tune: `group_size` and `ratio`. Please refer to the [example](link) how to automatically tune these parameters. 

```